src/router/router.tsx

```diff
@@ -1,14 +1,84 @@
 import { createBrowserRouter } from 'react-router-dom';
 import { HomePage } from '@ui/views';
+import { Overview } from '@ui/views/HowToPlay/Overview';
```

Comment:
This is a nitpick, so feel free to ignore.

One suggestion if you prefer clean up the imports:

- add an index file for the HowToPlay folder: `src/views/HowToPlay/index.ts`
- in that file, add `export { Overview } from './Overview'`

This will pass the component up the directory tree. You can do the same for "Lifelines" and import them from the same place

```suggestion
import { Overview, Lifelines } from '@ui/views/HowToPlay';
```